CHANGELOG.md

```diff
@@ -1,3 +1,197 @@
+# 6.2.0
+
+## Features
+
+### New processes
+
+- HeatTransportBHE process supporting 1U, CXAa, and CXC BHE types. #2221, #2271,
+  #2275
```

Comment:
I don't think there is a BHE type of CXAa, could you please confirm it again? Plus there is a PR regarding coaxial type BHE, #2332

Reply:
my typo. The corresponding PR is added.